demo/src/screens/incubatorScreens/WheelPickerScreen.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
-import {View, Text, Incubator} from 'react-native-ui-lib';
+import React, {useCallback} from 'react';
+import {View, Text, Incubator, Typography, Colors} from 'react-native-ui-lib';
 import _ from 'lodash';
+import { ItemProps } from 'src/incubator/WheelPicker/Item';
```

Comment:
make sure to format the code with prettier (the curly braces spacings are not according to our standards) 
I think eslint should have given you error for that